MagicalRecord/Categories/DataImport/NSEntityDescription+MagicalDataImport.h

```diff
@@ -7,11 +7,12 @@
 //
 
 #import <CoreData/CoreData.h>
+#import "MagicalRecordXcode7CompatibilityMacros.h"
```

Comment:
```#import ""``` won't be recognised by the framework users properly in case ```NSEntityDescription+MagicalDataImport.h``` is a public header


```obj-c
#import <MagicalRecord/MagicalRecordXcode7CompatibilityMacros.h>
```
That's the way Apple builds their umbrella headers.